New commands usage issues

Hi,

I encountered a few issues using the new commands:

### Can't get backups
```bash
clever database backups POSTGRESQL-ID
[ERROR] organisation ID is mandatory
```

So I added --org:
```bash
clever database backups --org OWNER-ID POSTGRESQL-ID
[ERROR] organisation ID is mandatory
```

### No indication
```bash
clever database
not available yet
```

Let us render the list of the available commands for the database, but maybe it's not accessible due to cliparse?

### Wrong indication
```bash
A database ID (format: postgres_UUID)
```

`s/postgres/postgresql` as Clever Cloud PostgreSQL Addon pattern.

Using:
```bash
clever diag
clever-tools  2.10.0
Node.js       v12.22.1
Platform      linux
Release       6.1.2+
Architecture  x64
Linux         Exherbo Linux
Packaged      true
Exec path     /usr/x86_64-pc-linux-gnu/bin/clever
User id       REDACTED
oAuth token   REDACTED
```

--- a/bin/clever.js
+++ b/bin/clever.js
@@ -1024,7 +1024,11 @@ function run () {
   const databaseCommand = cliparse.command('database', {
     description: 'List available databases',
     commands: [backupsCommand],
-  }, () => console.info('not available yet'));
+  }, () => {
+    console.info('This command is not available, you can try the following commands:');
+    console.info('clever database backups');
+    console.info('clever database backups download');
+  });
 
   // CLI PARSER
   const cliParser = cliparse.cli({

--- a/bin/clever.js
+++ b/bin/clever.js
@@ -89,7 +89,7 @@ function run () {
     }),
     appNameCreation: cliparse.argument('app-name', { description: 'Application name' }),
     backupId: cliparse.argument('backup-id', { description: 'A Database backup ID (format: UUID)' }),
-    databaseId: cliparse.argument('database-id', { description: 'A database ID (format: postgres_UUID)' }),
+    databaseId: cliparse.argument('database-id', { description: 'A database ID (format: postgresql_UUID, mysql_UUID, ...)' }),
     drainId: cliparse.argument('drain-id', { description: 'Drain ID' }),
     drainType: cliparse.argument('drain-type', {
       description: 'Drain type',

--- a/src/commands/database.js
+++ b/src/commands/database.js
@@ -20,6 +20,11 @@ async function listBackups (params) {
 
   const backups = await getBackups({ ownerId, ref: databaseId }).then(sendToApi);
 
+  if (backups.length === 0) {
+    println('There are no backups yet');
+    return;
+  }
+
   const formattedLines = backups
     .sort((a, b) => a.creation_date.localeCompare(b.creation_date))
     .map((backup) => [

--- a/src/commands/database.js
+++ b/src/commands/database.js
@@ -83,7 +83,7 @@ async function resolveOwnerId (org, databaseId) {
 
   const summary = await getSummary().then(sendToApi);
 
-  const userHasAddon = summary.user.addons.some((addon) => addon.readId === databaseId);
+  const userHasAddon = summary.user.addons.some((addon) => addon.realId === databaseId);
   if (userHasAddon) {
     return summary.user.id;
   }

